executors/dart_web/bin/likely_subtags.dart

```diff
@@ -15,21 +15,20 @@ String testLikelySubtags(String jsonEncoded) {
     String resultLocale;
     if (testOption == 'maximize') {
       resultLocale = locale.maximize().toLanguageTag();
-    } else if (testOption == 'minimizeFavorScript' ||
+    } else if (testOption == 'minimizeFavorRegion' ||
         testOption == 'minimize') {
       resultLocale = locale.minimize().toLanguageTag();
-    } else if (testOption == 'minimizeFavorRegion') {
+    } else if (testOption == 'minimizeFavorScript') {
       throw UnimplementedError();
```

Comment:
Should this be returned as an unsupported result type rather than throwing?

Reply:
As I understand it, the throw here is caught below, adding error_detail and unsupported to the result.

Reply:
Yep, I agree. Curious way to code that up.